packages/contracts/src/utils/types.ts

```diff
@@ -38,6 +38,8 @@ export interface DefaultOrderParams {
     takerTokenAmount: BigNumber;
     makerFeeAmount: BigNumber;
     takerFeeAmount: BigNumber;
+    makerAssetProxyId: number;
```

Comment:
Is there a better type to use than `number`? This will get downcast to `uint8`.

Reply:
In JS - no.

Reply:
Maybe a union type? 1|2|3?

Reply:
I would use a custom enum type:

```
enum AssetProxyIds {
  InValid,
  ERC20,
  ERC721,
  ...
}
```

These map to 0, 1, 2, ... so the first enum value should be set to `Invalid` if the id's start with `1` in the smart contracts.

Reply:
I like the Enum approach. It can match the Enum used in the contracts.